### PR TITLE
Visitor can change language

### PR DESCRIPTION
**Pull request by Jon1969Edwards**
## Pivotal Tracker
[Pivotal Tracker](https://www.pivotaltracker.com/story/show/163088245)

## Tasks
- Add a feature file to include language option
- Add links for language option in navbar
- Configure language options in config/application.rb
- Include categories to translate

## Gems
 'rails-i18n'

## Description/User Story
As a user
In order to be able to read articles in my native language,
I would like to switch between languages English and Swedish

screen shot 2019-01-19 at 12 34 02 pm

screen shot 2019-01-19 at 12 33 53 pm

### CODE REVIEW

**Review by CamronLDNF — APPROVED** [review]
Looks good to me. Go ahead.

**Review by johannaanden — APPROVED** [review]
LGTM

File: app/views/partials/_category_navbar.html.haml [hunk 1]
```diff
@@ -2,4 +2,4 @@
     - categories.each do |category|
         %li
             %p
-                = link_to category.name, category_path(category), class: 'category-item'
\ No newline at end of file
+                = link_to t(category.name), category_path(category), class: 'category-item'
```
Comment (oliverochman) [inline app/views/partials/_category_navbar.html.haml:5]: is the "t" in `t(category.name)` a typo?

File: features/visitor_can_view_all_listed_articles.feature [hunk 2]
```diff
@@ -12,8 +12,11 @@ Feature: Visitor can view all listed articles
...
             | title | description    | content               | user_id | category_id | published |
             | News  | This is a news | I like to eat cheese! | Bill    | Sport       | true      |
+      
 
     Scenario: Visitor can see articles on index page
         Given I visit the landing page
-        Then I should see 'News'
-        Then I should see 'This is a news'
\ No newline at end of file
+        And I should see 'News'
```
Comment (oliverochman) [inline features/visitor_can_view_all_listed_articles.feature:19]: You need to have a "Then" here

**Review by oliverochman — CHANGES_REQUESTED** [review]

File: app/views/partials/_category_navbar.html.haml (hunk 1 above)
Comment (Jon1969Edwards) [reply app/views/partials/_category_navbar.html.haml:5]: No.  We need this for changing the language to Swedish.

File: features/visitor_can_view_all_listed_articles.feature (hunk 2 above)
Comment (Jon1969Edwards) [reply features/visitor_can_view_all_listed_articles.feature:19]: This has been fixed.

File: app/views/partials/_category_navbar.html.haml (hunk 1 above)
Comment (oliverochman) [reply app/views/partials/_category_navbar.html.haml:5]: Okay, thanks for the reply

**Review by oliverochman — APPROVED** [review]

**Pull request merged**